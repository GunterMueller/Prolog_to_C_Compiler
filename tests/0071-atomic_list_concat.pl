main :-
	atomic_list_concat([abc,'123.0',45,9.1,'___'], X), writeq(X), nl.
