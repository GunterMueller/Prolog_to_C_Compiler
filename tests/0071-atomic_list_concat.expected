'abc123.0459.1___'
